src/features/search/components/__tests__/SearchResults.native.test.tsx

```diff
@@ -220,4 +232,53 @@ describe('SearchResults component', () => {
       expect(typeButtonLabel).toHaveStyle({ color: theme.colors.primary })
     }
   )
+
+  it('should not display geolocation incitation button when position is not null', async () => {
+    const { queryByText } = render(<SearchResults />)
+
+    await act(async () => {
+      expect(queryByText('Géolocalise toi')).toBeFalsy()
+    })
+  })
+
+  it.each`
+    filter                                                      | params
+    ${'digital offer'}                                          | ${{ offerTypes: { isDigital: true, isEvent: false, isThing: false } }}
```

Comment:
C'est un détail, mais tu n'as pas besoin de string compute ici
```suggestion
    digital offer.                                              | ${{ offerTypes: { isDigital: true, isEvent: false, isThing: false } }}
```

Reply:
Si je le retire ça casse le test

Reply:
C'est bizarre... C'est un string, non ? Je comprends pas trop comment c'est possible que ça casse :/
Mais soit, c'est du détail, on laisse tomber :D Faudra checker à l'occasion